new resource `azurerm_spring_cloud_api_portal`

ms-henglu opened the pull request:
![image](https://user-images.githubusercontent.com/79895375/167539412-e2578c29-80a9-4f3e-a345-9bb33c309928.png)

katbyte commented on website/docs/r/spring_cloud_api_portal.html.markdown:
```diff
@@ -0,0 +1,108 @@
+
+* `gateway_ids` - (Optional) Specifies a list of Spring Cloud Gateway.
+
+* `https_only` - (Optional) is only https is allowed?
```
shoudl this be?
```suggestion
* `https_only_enabled` - (Optional) is only https is allowed?
```

katbyte reviewed (changes requested):
test failures:
<img width="1670" alt="image" src="https://user-images.githubusercontent.com/1638467/168195026-0722ff24-a030-4964-802c-86f99e086f4f.png">

ms-henglu commented:
@katbyte - I've fixed the testcases, and renamed the field, please take another look, thanks!
![image](https://user-images.githubusercontent.com/79895375/168209159-2ce89ab4-a147-4839-9a33-32c85deb1d80.png)

katbyte reviewed (approved):
Thanks @ms-henglu - LGTM 🍾
